draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +448,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
 
-It is important to note that, from a functional perspective of coherent pluggables, the authority that defined the supported modes is less critical than the specific attributes associated with each supported mode. The key consideration is the list of attributes defined by these supported modes.
+* Standard Mode: This mode pertains to optical specifications developed by standards development organizations (SDOs), such as the ITU-T recommendation {{G.698.2}}.
+* Organizational Mode: In this mode, optical interface specifications are defined by operators, industry forums (e.g., Optical Internetworking Forum (OIF) or OpenConfig), or equipment vendors. This allows for the utilization of optical pluggable capabilities that extend beyond existing standards.
+* Explicit Mode: This mode enables the explicit encoding of any subset of parameters (e.g., FEC type, modulation type) to facilitate interoperability checks by a controller entity through means not covered within this draft.
 
-As illustrated in {{figure-plug-supported-mode}}, this document utilizes a consistent data structure for defining supported modes, irrespective of the authority that specified them. It includes:
+For more detailed information, please refer to draft {{ietf-impairment-yang}}.
 
-* supported-mode-id: Identification of the supported mode
-* organization-id: This provides a name space for an authority who defines the supported-mode, i.e., the authority that defines the supported-mode, such as ITU-T, OIF, OpenConfig, or a vendor
-* tags: Optional multiple tags that can be used for various purposes, mainly for future extensibility. For example, these tags can map the supported-mode to CMIS Media-ID (if they are not the same). Other potential uses of this tag are reserved for future developments.
-* operational-mode: This is the main part of this document which contains a list of capability attributes supported by the coherent pluggable.
+{: #plug-config-attribute}
+## Coherent Pluggable Configurations Attributes
 
-{{plug-manifest}} discusses the concept of the "coherent pluggable manifest", which is a repository for all supported-modes". It also outlines the benefit of such repository and various use-cases.
-Some details of supported modes can be found in IETF draft {{Impairment}}.
+Referring to {{figure-plug-config}}, the coherent pluggables support a set of read-write attributes which are configurable. Example of such configuration attributes are output power, central frequency and operational-mode. Note that as discussed in {{plug-capabilities-attributes}}, since a coherent pluggable may support multiple operational-modes, as part of these configuration attributes, operator should configure which of these operational-mode is desired and should be functional.
```

Comment:
line 462:
Referring to {{figure-plug-config}}, optical modules support a set of read-write attributes which are configurable. Example of such configuration attributes are output power, central frequency and operational-mode. Note that as discussed in {{plug-capabilities-attributes}}, since optical modules may support multiple operational-modes, as part of these configuration attributes, operator should configure which of these operational-mode is desired and should be functional.